spec/components/works/show/header_component_spec.rb

```diff
@@ -53,6 +53,21 @@
     end
   end
 
+  context 'when pending review but user is collection manager' do
+    let(:editable_version_status) { true }
+    let(:review_state) { 'pending_review' }
+
+    before do
+      work.collection.managers = [user]
+      work.collection.save!
+    end
+
+    it 'does not render the edit button' do
```

Comment:
test description seems like it might be copypasta, since the expectation passes if the button is there?

Reply:
Duh. Fixed.